torch/ao/quantization/_dbr/auto_trace.py

```diff
@@ -87,17 +88,28 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             if global_disable_torch_function_override:
                 return super().__torch_function__(func, types, args, kwargs)
 
-            # to prevent printing things from going into an infinite loop
             if func == torch.Tensor.__repr__:
+                # to prevent printing things from going into an infinite loop
                 return super().__torch_function__(func, types, args, kwargs)
+
+            # if we are in a function, the current module is always a parent
+            nonlocal cur_module
+            parent_module = cur_module
             if enable_logging:
-                logger.debug(f'__torch_function__ {str(func)} len_args {len(args)}')
+                is_obs_or_fq = isinstance(
```

Comment:
I remember we have a helper function for this: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/quantize.py#L29
should probably move this to torch.ao.quantization.utils